appservice/src/createAppService/AppInsightsCreateStep.ts

```diff
@@ -44,6 +45,13 @@ export class AppInsightsCreateStep extends AzureWizardExecuteStep<IAppServiceWiz
                     wizardContext.appInsightsComponent = await client.components.createOrUpdate(rgName, aiName, { kind: 'web', applicationType: 'web', location: appInsightsLocation });
                     const createdNewAppInsights: string = localize('createdNewAppInsights', 'Successfully created Application Insights resource "{0}".', aiName);
                     ext.outputChannel.appendLog(createdNewAppInsights);
+                } else if (pError.errorType === 'AuthorizationFailed') {
+                    if (!wizardContext.advancedCreation) {
+                        const createAppInsightsNotAvailable: string = localize('appInsightsNotAvailable', 'Skipping creating an Application Insights because the user doesn\'t have the right permissions.');
```

Comment:
Trying to match the other prompts/logs better:
```suggestion
                        const createAppInsightsNotAvailable: string = localize('appInsightsNotAvailable', 'Skipping Application Insights resource because you do not have permission to create one in this subscription.');
```

Reply:
I was trying to match this line (60), 
            `const appInsightsNotAvailable: string = localize('appInsightsNotAvailable', 'Skipping creating an Application Insights resource because it isn\'t compatible with this location.');<br class="Apple-interchange-newline">`. 
Should this be modified as well?

Reply:
Oh didn't see that one. Yeah how about we change that to:
> Skipping Application Insights resource because it isn\'t compatible with this location.


I just don't like "skipping" and "creating" right next to each other. Too many "ing"s